File: py/exerc007.py
#**********************************************************************
# FILE NAME   : EXERC007                                              *
# DATE        : 2025-03-26                                            *
# AUTHOR      : FABIO MARQUES (FMARQUES@FMARQUES.ETI.BR)              *
# DATA CENTER : COMPANY.EDUC360                                       *
# PURPOSE     : CALCULATE BMI (BODY MASS INDEX) AND DISPLAY STATUS    *
#**********************************************************************

# Variable declaration
peso   = 0
altura = 0
imc    = 0

# Reading values
peso   = float(input("Informe o peso:    "))
altura = float(input("Informe a altura:  "))

# Calculating BMI
imc = (peso / (altura * altura))

# Display BMI value
print("\nIMC: ", imc)

# Display status
if imc < 18.5:
    print("Abaixo")
elif imc >= 18.5 and imc < 24.9:
    print("Normal")
elif imc >= 25.0 and imc < 29.9:
    print("Sobrepeso")
elif imc >= 30.0 and imc < 39.9:
    print("Acima")

File: js/exerc007.js
/**********************************************************************
* FILE NAME   : EXERC007                                              *
* DATE        : 2025-03-29                                            *
* AUTHOR      : FABIO MARQUES (FMARQUES@FMARQUES.ETI.BR)              *
* DATA CENTER : COMPANY.EDUC360                                       *
* PURPOSE     : CALCULATE BMI (BODY MASS INDEX) AND DISPLAY STATUS    *
**********************************************************************/

// Variable declaration
let peso   = 0
let altura = 0
let imc    = 0

// Reading values
peso   = parseFloat(prompt("Informe o peso"))
altura = parseFloat(prompt("Informe a altura"))

// Calculating BMI
imc = (peso / (altura * altura))

// Display BMI value
console.log("IMC: " + imc)

// Display status
if (imc < 18.5) {
  console.log("Abaixo")
} else if (imc >= 18.5 && imc < 24.9) {
  console.log("Normal")
} else if (imc >= 25.0 && imc < 29.9) {
  console.log("Sobrepeso")
} else if (imc >= 30.0 && imc < 29.9) {
  console.log("Acima")
}
